R/bundleFiles.R

```diff
@@ -178,8 +178,6 @@ ignoreBundleFiles <- function(dir, contents) {
   ignore <- c(
     # rsconnect packages
     "rsconnect", "rsconnect-python", "manifest.json",
-    # packrat + renv,
-    "renv", "renv.lock", "packrat",
```

Comment:
Not sure if we should still ignore some of these? Or some parts of the renv/packrat directories?

Reply:
If we ignore them, they won't be included in the `bundleDir()` and `snapshotRDependencies()` won't see them.

Reply:
Based on further discussions we'll now ignore the `packrat/` and `renv/` directories automatically, and remove the `renv.lock` once we're done with it.